android/modules/ui/src/java/ti/modules/titanium/ui/widget/tableview/TiTableView.java

```diff
@@ -48,7 +48,8 @@
 	public static final int TI_TABLE_VIEW_ID = 101;
 	private static final String TAG = "TiTableView";
 
-	protected int maxClassname = 32;
+	// Default maxClassname is 32. It is set to 35 for bug in TIMOB-20038
+	protected int maxClassname = 35;
```

Comment:
since you already do +3, this doesn't need to be 35, 32 should be good, otherwise you'd add twice.